test/infrastructure/docker/config/webhook/manifests.yaml

```diff
@@ -1,11 +1,100 @@
 
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
```

Comment:
I think adding this requires additions to `webhookcainjection_patch.yaml`

Reply:
Are there example of the kinds of additions I need to do?

Also do you think this could this be the reason for the piepline failing with the following error:
```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "validation.dockercluster.infrastructure.cluster.x-k8s.io": Post "https://capd-webhook-service.capd-system.svc:443/mutate-infrastructure-cluster-x-k8s-io-v1alpha4-dockercluster?timeout=10s": x509: certificate signed by unknown authority
```

Reply:
Roughly: (we should also have it in the `webhookcainjection_patch.yaml` files of other controllers
```
apiVersion: admissionregistration.k8s.io/v1
kind: MutatingWebhookConfiguration
metadata:
  name: mutating-webhook-configuration
  annotations:
    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```